lib/path_manager.c

```diff
@@ -45,52 +45,104 @@ static void family_send_callback(struct l_genl_msg *msg, void *user_data)
 
 // ---------------------------------------------------------------------
 
-static size_t get_addr_size(struct mptcpd_in_addr const *addr)
+static inline bool is_inet_family(struct sockaddr const *addr)
```

Comment:
I was taking an extra look at this since `is_inet_family()` is only used in `assert`s, but I'm guessing it's inline to make the compiler not complain

Reply:
I was planning on using it in other areas of that file, but ended up not doing so.  The `inline` isn't strictly necessary since it's only used in an `assert()`.  I'll drop the `inline`.

Reply:
Nevermind.  I see what you're saying.  I haven't actually confirmed that the compiler will complain about an unused out-of-line static function.  Would you prefer that I simply remove the function given that it is only used in to two `assert()` calls?

Reply:
It's fine as-is with the inline. I noticed that it was only called in the asserts, and did a non-debug build to see what happened - figured you had intentionally made it inline to prevent the unused function warning :)